db/seeds.rb

```diff
@@ -37,28 +42,47 @@
 
 p 'creating classroom'
 
-Classroom.create!(
-  user_id: User.where(role: 1).first.id,
-  name: '4B English'
-)
-Classroom.create!(
-  user_id: User.where(role: 1).last.id,
-  name: '1C Math'
-)
+subjects = ['English', 'Maths', 'Science', 'Geography', 'History']
+groups = ['A', 'B', 'C']
```

Comment:
I see what you are going for here
actually it would be better if you would explicitly write the subject and group when creating the classrooms (instead of the name) for lines 42 and 46 in the old code

The way you are doing wouldn't actually work as other parts of the seeds would break (like StudentWorkGroup)

Reply:
I'm not sure what I need to change here. When I run the seed it works fine so I'm not sure what it would be breaking exactly.

Reply:
Just write it as 
```
Classroom.create!(
  user_id: User.where(role: 1).first.id,
  subject: 'English',
  group: 'A',
  grade: 5
)

Classroom.create!(
  user_id: User.where(role: 1).last.id,
  subject: 'Math',
  group: 'B',
  grade: 3
)
```

What you have "works" because we don't have validations in place (and I'm going to add them after this), but if you look at the code here, you could see that you could end up with a Student assigned to a WorkGroup that is part of a Classroom that the Student doesn't belong to: https://github.com/odaiba-app/server/pull/82/files#diff-f7a25bc30a409fc77050703fc048ca8cR120-R128